ios/Classes/SwiftCustomerIoPlugin.swift

```diff
@@ -93,87 +94,82 @@ public class SwiftCustomerIoPlugin: NSObject, FlutterPlugin {
     private func track(params : Dictionary<String, AnyHashable>)  {
         // TODO: Fix track implementation
         /*
-        guard let name = params[Keys.Tracking.eventName] as? String
-        else {
-            return
-        }
-        
-        guard let attributes = params[Keys.Tracking.attributes] as? Dictionary<String, AnyHashable> else{
-            CustomerIO.shared.track(name: name)
-            return
-        }
-        
-        CustomerIO.shared.track(name: name, data: attributes)
+         guard let name = params[Keys.Tracking.eventName] as? String
```

Comment:
Is it possible to revert these formatting changes for commented code? As it could be a bit challenging when merging as I'm working in same file in other ticket.